index.html

```diff
@@ -0,0 +1,93 @@
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+    <link rel="icon" href="./images/apple-touch-icon-57x57.png" type="image/png" sizes="16x16">
+    <link rel="stylesheet" href="./index.css">
+    <title>Intuit Accounts - Sign Up</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="nav-right-column">
+                <div class="description">
+                    <div class="fade">Already have an Intuit account?</div>
+                    <a href="#">I forgot my User ID or Password</a>
+                </div>
+                <button type="button">
+                    <i class="material-icons icons" style="font-size: 14px;">lock</i>
+                    Sign In
+                </button>
+            </div>
+            <div class="nav-left-column"></div>
+        </nav>
+    </header>
+    <main>
+        <aside class="col-1"></aside>
+        <section class="container">
+            <div class="container-header">
+                <div class="banner"></div>
+                <ul class="menu-sprite">
+                    <li class="icon logo-1"></li>
+                    <li class="sprint mint"></li>
+                    <li class="icon logo-2"></li>
+                    <li class="sprint quickbook-logo"></li>
+                    <li class="icon logo-3"></li>
+                    <li class="sprint turbotax"></li>
+                </ul>
+            </div>
+            <div class="form-container">
+                <div class="form-container-header">
+                    <h2>Create an Intuit account</h2>
+                    <div>One account for everything Intuit, including Mint</div>
+                    <a href="#">Learn more</a>
+                </div>
+                <form action=".">
+                    <div class="form-control">
+                        <label for="email">Email address</label>
+                        <input type="email" name="email" id="email">
+                    </div>
+                    <div class="form-control">
+                        <label for="phone">Phone (recommended)</label>
+                        <input type="number" name="phone" id="phone">
+                        <div class="input-info-text">Standard call, messaging or data rates may apply.</div>
+                    </div>
+                    <div class="form-control">
+                        <label for="pwd">Password</label>
+                        <input type="password" name="pwd" id="pwd">
+                    </div>
+                    <div class="form-control hide">
+                        <label for="pwd2">Confirm password</label>
+                        <input type="password" name="pwd2" id="pwd2">
+                    </div>
+                    <div class="form-control">
+                        <button type="submit">
+                            <i class="material-icons icons" style="font-size: 14px;">lock</i>
```

Comment:
## Using Inline Styles[1]

- Avoid using inline styles. 

- [ ] Please move to your CSS file all your `style=""` properties



###### _[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_